ee/clickhouse/queries/session_recordings/test/test_clickhouse_session_recording_list.py

```diff
@@ -20,4 +24,61 @@ def _create_session_recording_event(**kwargs):
 
 
 class TestClickhouseSessionRecordingsList(ClickhouseTestMixin, factory_session_recordings_list_test(ClickhouseSessionRecordingList, _create_event, _create_session_recording_event, Action.objects.create, ActionStep.objects.create)):  # type: ignore
-    pass
+    def test_event_filter_with_person_properties(self):
```

Comment:
Could we leverage `@test_with_materialized_columns` here to make sure we make use of mat columns?

Reply:
I added this, but because the `session_recording_events` part of the query is still using JSONExtract, I had to set `verify_no_jsonextract=False`. I might be wrong, but I don't think the `session_recording_events` would ever have materialized columns - maybe we want to change this?

Reply:
Using `verify_no_jsonextract` is completely fair here :) Thanks for adding it.